packages/api/lib/writeRecords/write-granules.js

```diff
@@ -869,8 +876,13 @@ const writeGranulesFromMessage = async ({
       const updatedAt = now;
       const timestamp = now;
 
+      let published = granule.published;
+      if (isNil(published)) {
+        published = false;
```

Comment:
Just out of curiosity, is there a reason that `writeGranuleFromApi`'s published check doesn't use `isNil`?